docker/driver-loader-legacy/docker-entrypoint.sh

```diff
@@ -16,18 +16,13 @@
 # limitations under the License.
 #
 
-# Set the SKIP_DRIVER_LOADER variable to skip loading the driver
 
-if [[ -z "${SKIP_DRIVER_LOADER}" ]]; then
```

Comment:
Why this change?

Reply:
the `driver-loader` image is different from `falco`. In fact, it has a different entrypoint! This makes sure that the change is reflected, so that the `driver-loader-legacy` image actually behaves like a `driver-loader` (as in, it loads the driver) and not like a `falco` image (which loads the driver AND starts Falco)